tests/test_django_cloudtask.py

```diff
@@ -1,10 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import absolute_import, unicode_literals
-
-from django.test import TestCase
-from django_cloudtask import __version__
-
-
-class DjangoCloudtaskTestCase(TestCase):
```

Comment:
Curious why you wanted this test in the first place -- just a sanity check on version number for release?